tests/tests/fv3_gfs_v16_RRTMGP

```diff
@@ -75,10 +87,14 @@ export LIST_FILES="atmos_4xdaily.tile1.nc \
 export_fv3
 
 DT_ATMOS="1200"
+export FHMAX=48
```

Comment:
May I ask why the forecast time is increase from 24 to 48hr? Will 24hr work?

Reply:
I did this to be consistent with the fv3_gfs_v16 RT.

Reply:
fv3_gfs_v16 has this to prepare restart test. I am updating global regression tests, so the RRTMGP test will also be updated, will ask you to check when the PR is ready.